data/json/items/comestibles/med.json

```
@@ -1677,6 +1677,24 @@
     "flags": [ "IRREPLACEABLE_CONSUMABLE" ],
     "charges": 10,
     "stack_size": 10
+  },
+    {
+    "id": "sugar_pills",
+    "type": "COMESTIBLE",
+    "comestible_type": "MED",
+    "name": "homeopathic pill",
+    "description": "Some homeopathic pills, prepared through the repeated dilution of an active ingredient. The label purpots that the pills are able to treat flu-like symptoms and maintain general wellness.",
```

```suggestion
    "description": "Some homeopathic pills, prepared through the repeated dilution of an active ingredient.  The label purpots that the pills are able to treat flu-like symptoms and maintain general wellness.",
```
Double spaces after a ful stop